tutorials/lwr_amr/openmc_amr.i

```diff
@@ -0,0 +1,98 @@
+[Mesh]
```

Comment:
Can we add some tests for these tutorials? Mostly for the purpose to catch any deprecated inputs/etc., since our regression test suite already covers all the AMR capabilities.